App/Sources/Presentation/StudentMain/Source/Scene/StudentMainView.swift

```diff
@@ -1,9 +1,177 @@
+                                    .cornerRadius(10)
+                                    .frame(height: 158)
+                                    
+                                    
+                                    if ranking == 1 {
+                                        Circle()
+                                            .frame(width: 20, height: 20)
+                                            .foregroundColor(Color.SKColorSystem.Medal.firstplace.color)
+                                            .overlay(
+                                                Text("1")
+                                                    .skFont(.pm14)
+                                            )
+                                    }
+                                    else if ranking == 2 {
+                                        Circle()
+                                            .frame(width: 20, height: 20)
+                                            .foregroundColor(Color.SKColorSystem.Medal.secondplace.color)
+                                            .overlay(
+                                                Text("2")
+                                                    .skFont(.pm14)
+                                            )
+                                    }
+                                    else if ranking == 3 {
+                                        Circle()
+                                            .frame(width: 20, height: 20)
+                                            .foregroundColor(Color.SKColorSystem.Medal.thirdplace.color)
+                                            .overlay(
+                                                Text("3")
+                                                    .skFont(.pm14)
+                                            )
+                                    }
+                                }
+                            }
+                        }
+                    }
+                    .padding(.vertical, 16)
+                    .padding(.horizontal, 16)
+                    .background(Color.SKColorSystem.Gray.lightgray1.color)
+                    .cornerRadius(20)
+                }
+                .padding(.top, 10)
+                .padding(.horizontal, 16)
+                
+                .logoNavigationBar()
+            }
+            .overlay(
+                Button(action: {
+                    showPopUp = true
+                }) {
+                    Image(systemName: "person.2")
+                        .resizable()
+                        .foregroundColor(.white)
+                        .padding(16)
+                        .background(Circle().fill(Color.SKColorSystem.Main.mainColor.color))
+                        .frame(width: 60, height: 60)
+                }
+                    .padding(.trailing, 30)
+                    .padding(.bottom, 74),
+                alignment: .bottomTrailing
+            )
```

Comment:
```suggestion
            .overlay(alignment: .bottomTrailing) {
                Button {
                    showPopUp = true
                } label: {
                    Image(systemName: "person.2")
                        .resizable()
                        .foregroundColor(.white)
                        .padding(16)
 .background(Circle().fill(Color.SKColorSystem.Main.mainColor.color))
                        .frame(width: 60, height: 60)
                        .padding(.trailing, 30)
                        .padding(.bottom, 74)
                }
```
이런 느낌으로 더 가독성있게 작성하면 좋을 것 같아요~!

Reply:
오호~ 바로 수정하겠습니다~ 다음에 비슷하게 작성하는 일이 생긴다면 위와 같이 작성해보도록 하겠습니다~